drivers/gpio/Kconfig.tca6424a

```diff
@@ -0,0 +1,19 @@
+# TCA6424A configuration options
+
+# Copyright (c) 2022 Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GPIO_TCA6424A
+	bool "TCA6424A driver"
+	default y
+	depends on DT_HAS_TI_TCA6424A_ENABLED
+	depends on I2C
+	help
+	  Enable driver for TCA6424A IO expander chip.
+
+config GPIO_TCA6424A_INIT_PRIORITY
```

Comment:
I'll just mention the same thing that I mentioned in https://github.com/zephyrproject-rtos/zephyr/pull/51741, which is that init priority can actually become slightly more complicated for off-chip GPIO controllers. 

On systems where there are more than one of these GPIO controllers, and determinism is somewhat important (i.e. certain output pins need to come up in a certain order), it could be helpful to make a per-instance `init-priority` property in DT which can default to a common Kconfig value.

E.g. 
```
DT_INST_PROP_OR(inst, init_priority, GPIO_TCA6424A_INIT_PRIORITY)
```